### ISSUE TITLE
Align text and number into Content

### ISSUE THREAD

**Issue by Seby02** [QuestPDF/QuestPDF#894]
Hello,
Thanks for QuestPDF, it's an amazing tool

i have a question about alignment, i would like to align my text like this :

![Capture d'écran 2024-05-16 135530](https://github.com/QuestPDF/QuestPDF/assets/33393241/220b895f-34d7-4d67-97bf-b809f498f101)

here my code :

container.PaddingVertical(40).Column(column =>
{
    column.Spacing(5);

    column.Item().Row(row =>
    {
        row.RelativeItem().Component(new AddressComponent("Vendeur", sellerAddress));
        row.ConstantItem(50);
        row.RelativeItem().Component(new AddressComponent("Acheteur", billingAddress));
    });

    column.Item().Element(x =>
    {
        ComposeTable(x, invoice);
    });

    if (invoice.vat_rate != 0)
    {
        // Calcul du montant de TVA
        vatAmount = (double)invoice.total_price * (double)invoice.vat_rate / 100.0;

        // Ajout du montant de TVA au total
        invoice.total_price = (double)invoice.total_price + vatAmount;
    }
    vatAmount = invoice.total_price;
    column.Item().AlignRight().AlignMiddle().PaddingRight(20).Text($"Total HT: {invoice.total_price.ToString("N2", cultureInfo)} €").FontSize(12);
    column.Item().AlignRight().AlignMiddle().PaddingRight(20).Text($"Taux de TVA : {invoice.vat_rate} %").FontSize(12);
    column.Item().AlignRight().AlignMiddle().PaddingRight(20).Text($"Total TVA : {vatAmount.ToString("N2", cultureInfo)} €").FontSize(12);
    column.Item().AlignRight().AlignMiddle().PaddingRight(20).Text("---------------------").FontSize(12);
    column.Item().AlignRight().AlignMiddle().PaddingRight(20) .Text($"Total TTC: {invoice.total_price.ToString("N2", cultureInfo)} €").Style(titleStyle);

    if (!string.IsNullOrWhiteSpace(Placeholders.LoremIpsum()))
        column.Item().PaddingTop(25).Element(ComposeComments);
});`

i test with alignMiddle of something like this but i don't have any results :)
sorry if you have already a reply about this :) 
Thanks in advance

**Comment by MarcinZiabek** [comment]
I usually recommend making an educated guess regarding the content size and dividing it into separate columns and rows. Page size is constant, so using concrete size values often allows us to achieve more stable layouts.

```csharp
var content = new[]
{
    ("Label 1", "Value 234"),
    ("Label 12", "Value 34"),
    ("Label 123", "Value 4")
};

container
    .Padding(20)
    .AlignRight()
    .Column(column =>
    {
        foreach (var value in content)
        {
            column.Item().AlignRight().Row(row =>
            {
                row.AutoItem().Text($"{value.Item1}:").Bold();
                row.ConstantItem(75).AlignRight().Text(value.Item2);
            });
        }
    });
```

<img width="676" alt="image" src="https://github.com/QuestPDF/QuestPDF/assets/9263853/5d8dac34-1d4c-49e3-8415-b825f6586e5c">

Alternatively, you can use some monospaced font and space-pad your text accordingly. For such tasks, it is best to use the non-break space character: https://github.com/QuestPDF/QuestPDF/issues/896

**Comment by kavatari** [comment]
I think this is a use case for a simple table.

**Comment by MarcinZiabek** [comment]
Indeed, this requirement can be implemented with multiple different approaches. The table method suggested by @kavatari will work nicely too 😄 

Since there has been no recent activity, I’m closing this issue. If you have a similar request in the future, feel free to open a new GitHub issue. Thanks for your feedback! ❤️
